robottelo/common/helpers.py

```diff
@@ -177,16 +179,24 @@ def invalid_names_list():
     List of invalid names for input testing.
     """
     return [
-        gen_string("alpha", 300),
-        gen_string("numeric", 300),
-        gen_string("alphanumeric", 300),
-        gen_string("utf8", 300),
-        gen_string("latin1", 300),
-        gen_string("html", 300),
-        gen_string("alpha", 256)
+        gen_string('alphanumeric', 300),
+        gen_string('alpha', 300),
+        gen_string('cjk', 300),
+        gen_string('latin1', 300),
+        gen_string('numeric', 300),
+        gen_string('utf8', 300),
+        gen_string('html', 300),
     ]
 
 
+def invalid_values_list():
+    """List of invalid values for input testing which consists from invalid names
+    and few empty string examples
```

Comment:
Invalid values... for what? Can you try to fiddle with this docstring and better explain what it does and what it can be used for? For example: where can these invalid values be used? As an argument to any command-line flag that takes an argument? When should a user choose to use this method instead of `invalid_names_list()`?